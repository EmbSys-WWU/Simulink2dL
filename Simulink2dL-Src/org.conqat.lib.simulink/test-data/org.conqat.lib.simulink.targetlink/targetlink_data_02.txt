struct('comment',struct('text','','placement',1),'logdata',struct('maxnumvals',inf,'plotchannels',[],'loggingmode',3,'channels',12),'output',struct('variable','','description','','name','$S_$B','width',12,'address','','max',NaN*ones(1,12),'min',NaN*ones(1,12),'class','default','type','uint8','scaling','','lsb',ones(1,12),'offset',zeros(1,12),'arb',zeros(1,12),'freeze',1,'unit','','autoscalingmode',1,'lowerheadroom',1,'upperheadroom',1,'headroomunit','bit','keepdatatype',1,'checkmin',0,'checkmax',0,'castoutput',0,'show',0),'threshold',struct('variable','','description','','name','$S_$B_threshold','width',12,'address','','max',NaN*ones(1,12),'min',NaN*ones(1,12),'class','default','type','Int16','scaling','','lsb',ones(1,12),'offset',zeros(1,12),'arb',zeros(1,12),'freeze',1,'unit','','keepdatatype',1,'value','1','scalar',0,'criteria',1),'scalingvalid',0,'date',733066.588839965)