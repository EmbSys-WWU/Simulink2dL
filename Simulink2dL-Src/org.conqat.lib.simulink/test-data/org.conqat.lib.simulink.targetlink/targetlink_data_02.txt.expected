struct (
  'comment' -> struct (
    'placement' -> '1'
    'text' -> ''
  )
  'logdata' -> struct (
    'channels' -> '12'
    'loggingmode' -> '3'
    'maxnumvals' -> 'inf'
    'plotchannels' -> '[]'
  )
  'output' -> struct (
    'address' -> ''
    'arb' -> 'zeros(1,12)'
    'autoscalingmode' -> '1'
    'castoutput' -> '0'
    'checkmax' -> '0'
    'checkmin' -> '0'
    'class' -> 'default'
    'description' -> ''
    'freeze' -> '1'
    'headroomunit' -> 'bit'
    'keepdatatype' -> '1'
    'lowerheadroom' -> '1'
    'lsb' -> 'ones(1,12)'
    'max' -> 'NaN*ones(1,12)'
    'min' -> 'NaN*ones(1,12)'
    'name' -> '$S_$B'
    'offset' -> 'zeros(1,12)'
    'scaling' -> ''
    'show' -> '0'
    'type' -> 'uint8'
    'unit' -> ''
    'upperheadroom' -> '1'
    'variable' -> ''
    'width' -> '12'
  )
  'threshold' -> struct (
    'address' -> ''
    'arb' -> 'zeros(1,12)'
    'class' -> 'default'
    'criteria' -> '1'
    'description' -> ''
    'freeze' -> '1'
    'keepdatatype' -> '1'
    'lsb' -> 'ones(1,12)'
    'max' -> 'NaN*ones(1,12)'
    'min' -> 'NaN*ones(1,12)'
    'name' -> '$S_$B_threshold'
    'offset' -> 'zeros(1,12)'
    'scalar' -> '0'
    'scaling' -> ''
    'type' -> 'Int16'
    'unit' -> ''
    'value' -> '1'
    'variable' -> ''
    'width' -> '12'
  )
  'date' -> '733066.588839965'
  'scalingvalid' -> '0'
)